TypeScript.py

```diff
@@ -1,19 +1,13 @@
 import sys
+import os
 
-# Sublime/Python 2 & 3 differ in the name of this module, thus package import
-# needs to be handled slightly differently
-if sys.version_info < (3, 0):
-    from libs import *
-    from libs.reference import *
-    from libs.view_helpers import *
-    from listeners import *
-    from commands import *
-else:
-    from .libs import *
-    from .libs.reference import *
-    from .libs.view_helpers import *
-    from .listeners import *
-    from .commands import *
+sys.path.append(os.path.dirname(os.path.abspath(__file__)))
```

Comment:
Given it another try, now it indeed worked. I think the reason is that last time I tried I didn't put everything under the same root folder. I'll test a bit more and update soon.